README.md

```diff
@@ -20,37 +20,16 @@ Please join the discussion on the Angle Community [Discord Server](https://disco
 
 ## Introduction to Angle
 
-### 🏅 Over-Collateralized, Decentralized, and Capital-Efficient Stablecoin Protocol
+Angle is a decentralized stablecoins protocol composed of smart contracts mostly deployed on the Ethereum blockchain. 
 
-Angle is the first over-collateralized, decentralized and capital-efficient stablecoin protocol. It offers full convertibility between stable assets and collateral at oracle value: you can swap 1 of collateral against 1 of stablecoin and conversely.
-
-This makes the protocol both capital efficient and highly liquid.
+It has three modules, or set of smart contracts: [Angle Core module](/concepts/overview.md), the new [Angle Borrowing module](/new-module/README.md), and the [Angle Governance module](/governance/angle-dao.md). 
```

Comment:
I don't think this is the first thing we want to put at the forefront. This is the first thing many people are going to see and if you lack context it is not super explanatory. 

We can put this sentence a bit like what we did here but not in first https://developers.angle.money/#protocol-design